src/mongo/util/net/ssl_options_client.idl

```diff
@@ -106,14 +105,7 @@ configs:
         deprecated_short_name: sslFIPSMode
         arg_vartype: Switch
         cpp_varname: "sslGlobalParams.sslFIPSMode"
-        requires:
-            expr: 'kFIPSModeEnabled ? "tls" : ""'
-            is_constexpr: true
-        hidden:
-            expr: "!kFIPSModeEnabled"
-            is_constexpr: true
-        validator:
-            callback: validateSslFIPSMode
+        requires: tls
```

Comment:
Looks like we don't need anymore changes introduced in #1575 since this option is now unconditional. I think we can revert them?

Reply:
Created a [task](https://perconadev.atlassian.net/browse/PSMDB-1877) for that.